dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -757,7 +757,7 @@ def get_airflow_extras():
     "3.10": "3.10.19",
     "3.11": "3.11.14",
     "3.12": "3.12.12",
-    "3.13": "3.13.9",
+y    "3.13": "3.13.10",
```

Comment:
Uuups with `y` :-D

Reply:
This is really strange :D I used `breeze ci upgrade` e2e. It seems when I press y and enter, this appears somehow in the file, then because Breeze even deleted my local branch, it is amazingly automated :D Fixing now

Reply:
Removed, checking the breeze command :)